data/json/recipes/weapon/bashing.json

```diff
@@ -829,6 +829,21 @@
     ],
     "components": [ [ [ "bo", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "i_staff",
+    "id_suffix": "with sheet metal",
+    "category": "CC_*",
+    "subcategory": "CSC_*_NESTED",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": "30 m",
+    "autolearn": true,
+    "proficiencies": [ { "proficiency": "prof_metalworking" } ],
+    "components": [ [ [ "sheet_metal_small", 2 ] ], [ [ "nuts_bolts", 4 ] ], [ [ "bo", 1 ] ] ],
```

Comment:
how big is a sheet metal small?

Reply:
Anywhere from 15x15cm to 20x20cm, which is more than large enough for an end cap I think since they're not meant to be mace heads. I also checked the weight; a small metal sheet weighs 0.25kg and the ironshod staff is only 0.25kg heavier than a bo staff, so overall this actually loses a bit of weight (presumably from just the extra bits being cut down).

Reply:
I have doubts that is enough metal to make a quarterstaff do more damage?

Reply:
The same could be said for the other recipe, then. Also, my guess is the weight for the ironshod was originally based off the weight for the regular quarterstaff which it originally used as the component, which only weighs 0.61kg.

It's probably the ironshod needs a weight increase overall, or maybe the bo staff just weighs too much, but that's probably out of scope for the PR. Otherwise, if more weight is needed for this particular crafting recipe, I could just increase the amount of small metal sheets to 6 or so under the assumption they're being layered for the end caps.

Reply:
I increased the small metal sheet cost from 2 to 6.